### ISSUE TITLE
SQL-Watch does not count secondary not readable replicas

### ISSUE THREAD

**Issue by hvoges** [marcingminski/sql-performance-monitor#461]
**Describe the bug**
SQL-Watch does not count non-readable Replicas on Always On Clusters. 

**To Reproduce**
Run select * from dbo.sqlwatch_meta_database on a secondary Replica with offline Replicas. There are no Entries for the Replicas, and as a result, they are not accounted for. 

**Expected behavior**
Mirrored Offline-Databases should be counted, especially as they use diskspace. 

**Screenshots**
![image](https://user-images.githubusercontent.com/6452224/194306154-4de71568-4d35-4f0a-8e45-5646099c2227.png)

**Windows Server (please complete the following information):**
 - OS Version: Windows Server 2016

**SQL Server (please complete the following information):**
 - SQL Version: SQL Server 2017
 - SQL Edition: Enterprise
 
**SQL Server Management Studio (SSMS -> about -> copy info):**
SQL Server Management Studio						15.0.18424.0
SQL Server Management Objects (SMO)						16.100.47021.0+7eef34a564af48c5b0cf0d617a65fd77f06c3eb1
Microsoft Analysis Services-Clienttools						15.0.19750.0
Microsoft Data Access Components (MDAC)						10.0.14393.0
Microsoft MSXML						3.0 6.0 
Microsoft .NET Framework						4.0.30319.42000
Betriebssystem						10.0.14393
 
**SQLWATCH version (from DACPAC or from sysinstances)**
 4.4.0.523

**Comment by marcingminski** [comment]
Thanks for this. This behaviour was introduced by this fix: https://github.com/marcingminski/sqlwatch/issues/107, and its currently by design. I suppose we could marry those two together.

**Comment by hvoges** [comment]
Thank you for your fast answer. The behaviour is a bit confusing as the count in the dashboard does not show the actual count.

**Comment by marcingminski** [comment]
I agree with you. I will keep this open and will think about how to address this.
